### PR TITLE
feature: Finish mobile version

### PR DESCRIPTION
**Pull request by Arsenethierry**
## Changes made:
in this pr changes, I added mobile sections for portifolio as recommended.

1. Added hero section
2. I added my previous projects section
3. I added who I am section
4. I added my contact or contact me section

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -0,0 +1,389 @@
...
+  background-color: white;
+  margin-left: 1.2em;
+  margin-right: 1.2em;
+  justify-self: center;
+  margin-top: 100px;
+  border-radius: 20px;
+  border: 1px solid #dfe1e6;
+  padding-bottom: 20px;
+}
+
```
Comment (pinkmoon25) [inline styles.css:111]: - [ ] great work so far you have done rrally well :clap: , Just few fixes and we are good to go, so as mentioned in the project requirements we have to make sure each project cards layout is done with Flexbox, flex is a crucial skill to master and it makes responsive design so much better, please use flex for the cards layout
image

File: styles.css [hunk 2]
```diff
@@ -0,0 +1,389 @@
...
+.last-btn {
+  border: 1px solid #6070ff;
+  padding: 10px 18px;
+  border-radius: 10px;
+  margin-top: 8px;
+  background-color: white;
+  color: #6070ff;
+  font-size: 20px;
+  font-weight: 600;
+}
```
Comment (pinkmoon25) [inline styles.css]: - [ ] Also we have to implement the button interactions for all the buttons on our page, and the font-size for these butoons should be 17px and font-weight 500 according to figma please correct that currently it is 20 px and 600 font-weight
image
 
here is the button interactions design from figma :point_down: 
image

please do this for all the buttons on your page

File: styles.css [hunk 3]
```diff
@@ -0,0 +1,389 @@
...
+
+.items {
+  padding-left: 20px;
+  margin-right: 50px;
+}
+
+.items h1 {
+  font-size: 2em;
+}
```
Comment (pinkmoon25) [inline styles.css]: - [ ] The color for the project cards title should be `#172B4D` according to the figma design
- [ ] [OPTIONAL] For better semantics I would recommend to not use an `h1` tag here but use use heading levels (`h2`, `h3`, etc.) . as generally, we use only a single `h1` tag per section this helps to create a clear and organized structure for the content on the page and makes it easier for search engines to understand the hierarchy and relevance of the content.

File: styles.css [hunk 4]
```diff
@@ -0,0 +1,389 @@
...
+
+.items p {
+  font-size: 16px;
+  text-align: start;
+  margin-right: 25px;
+  margin-bottom: 10px;
+}
```
Comment (pinkmoon25) [inline styles.css]: - [ ] The color for the project cards paragraph is supposed to be `#344563` according to figma

File: styles.css [hunk 5]
```diff
@@ -0,0 +1,389 @@
...
+  margin-bottom: 195px;
+  width: 100%;
+  display: flex;
+  flex-direction: column;
+}
+
+.about {
+  font-size: 2.4em;
+  padding-bottom: 10px;
+}
```
Comment (pinkmoon25) [inline styles.css]: - [ ] The color of the About me heading should be `#172B4D` according to figma

File: styles.css [hunk 6]
```diff
@@ -0,0 +1,389 @@
...
+
+.about-div p {
+  padding-bottom: 15px;
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  font-size: 16px;
+}
```
Comment (pinkmoon25) [inline styles.css]: - [ ] The color for about myself paragraph should be `#344563`

File: index.html [hunk 7]
```diff
@@ -0,0 +1,300 @@
...
+            <h5 class="m-l">CSS</h5>
+          </div>
+          <div class="languages-div skills">
+            <img src="images/Ellipse 1 (2).png" />
+            <h5 class="m-l">HTML</h5>
+          </div>
+        </div>
+        <ul class="languages-div framework">
+          <li>Framework</li>
+          <li><img src="images/union 2.png" /></li>
```
Comment (pinkmoon25) [inline index.html]: - [ ] For the Languages section we have to make sure the technology categories ("Languages", "Frameworks" and "Skills") are in a ul, each of them in a li also the technology in each category(e.g., "JavasScript", "HTML" and "CSS") are also a list ul and each of them a li, like :point_down: 
```
 <ul>
      <li> Language</li>
        <ul>
          <li>Javascript</li>
          <li>Ruby</li>
          <li>Html</li>
          <li>Css</li>
        </ul>
```

File: README.md [hunk 8]
```diff
@@ -1 +1,147 @@
...
+If you like this project please follow me on github & twitter and also connect on Linkedin.
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## :pray: Acknowledgments <a name="acknowledgements"></a>
+I would like to thank Microverse for this exercise.
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](https://github.com/Arsenethierry/portifolio-microverse/blob/feature/LICENSE.txt) licensed.
```
Comment (pinkmoon25) [inline README.md]: - [ ] Please fix your MIT link as currently it is broken and giving a 404 error

**Review by pinkmoon25 — APPROVED** [review]
Hi @Arsenethierry , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
Linter checks :heavy_check_mark: 
github flow :heavy_check_mark: 
Grid :heavy_check_mark: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by emmiiorji — APPROVED** [review]
Hi @Arsenethierry, 

# Status: Approved ✅

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

<div align='center'>

</div>

## Highlights

- Good design
- Project is well documented in readme

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**